project_testing/housing_characteristics/Lighting.tsv

```diff
@@ -1,2 +1,2 @@
-Option=100% Incandescent	Option=60% CFL Hardwired, 34% CFL Plugin	Option=60% LED Hardwired, 34% CFL Plugin	Option=60% CFL	Option=100% LED	Option=100% LED, Low Efficacy	Option=100% LED, Holiday Lights	Option=20% LED
-0.125	0.125	0.125	0.125	0.125	0.125	0.125	0.125
+Option=100% Incandescent	"Option=60% CFL Hardwired, 34% CFL Plugin"	"Option=60% LED Hardwired, 34% CFL Plugin"	Option=60% CFL	Option=100% LED	"Option=100% LED, Low Efficacy"	"Option=100% LED, Holiday Lights"	Option=20% LED	Option=NA

```

Comment:
I would call it "None" instead of "NA", similar to the Plug Loads and appliance parameters. Also, don't you have to add it to options_lookup.tsv?